### PR TITLE
fix: Updated to regex to include emoji

### PR DESCRIPTION
**Pull request by purnimavenkatasubbu**
**- What I did**
- Updated the keys verb regex. 
- modified keys verb builder

**- How I did it**
- Changed the regex to support emoji in the keyname
- modified keys verb builder to match ordering of param in syntax
**- How to verify it**
syntax_test.dart and keys_verb_builder_test.dart should pass
keys_verb_test.dart in the at_server function test should pass.
https://github.com/atsign-foundation/at_server/pull/1463

**- Description for the changelog**

### CODE REVIEW

File: packages/at_commons/lib/src/verb/keys_verb_builder.dart [hunk 1]
```diff
@@ -37,22 +37,23 @@ class KeysVerbBuilder implements VerbBuilder {
...
+          _getValue(value)) //value is prepended with a whitespace as per regexs
       ..write('\n');
 
     return sb.toString();
   }
 
   String _getValue(String? paramValue) {
     if (paramValue != null && paramValue.isNotEmpty && paramValue != 'null') {
-      return ':$paramValue';
+      return ' $paramValue'; //value is prepended with a whitespace as per regexs
```
Comment (murali-shris) [inline packages/at_commons/lib/src/verb/keys_verb_builder.dart]: typo..please change regexs to regex

**Comment by murali-shris** [comment]
please add unit test to cover emojis

File: packages/at_commons/lib/src/verb/keys_verb_builder.dart [hunk 2]
```diff
@@ -37,22 +37,23 @@ class KeysVerbBuilder implements VerbBuilder {
...
       ..write(_getValueWithParamName('namespace', namespace))
       ..write(_getValueWithParamName('appName', appName))
-      ..write(_getValueWithParamName('device', device))
+      ..write(_getValueWithParamName('deviceName', deviceName))
       ..write(_getValueWithParamName('keyType', keyType))
       ..write(_getValueWithParamName('encryptionKeyName', encryptionKeyName))
-      ..write(_getValue(value))
+      ..write(_getValueWithParamName('keyName', keyName))
+      ..write(
+          _getValue(value)) //value is prepended with a whitespace as per regexs
```
Comment (murali-shris) [inline packages/at_commons/lib/src/verb/keys_verb_builder.dart]: typo..please change regexs to regex

Comment (purnimavenkatasubbu) [reply packages/at_commons/lib/src/verb/keys_verb_builder.dart]: done

File: packages/at_commons/lib/src/verb/keys_verb_builder.dart (hunk 1 above)
Comment (purnimavenkatasubbu) [reply packages/at_commons/lib/src/verb/keys_verb_builder.dart]: updated

**Review by murali-shris — APPROVED** [review]

**Pull request merged**